README.md

```diff
@@ -11,47 +11,41 @@
 
 This documentation describes the process to apply and validate an IoT Device as FIWARE Ready.
 
-To join FIWARE-ready IoT Devices Program, you need to follow a series of steps, including an simple validation of your
-product (implementation), where you will connect it to a FIWARE public instance and you will perform a series of tests:
+To be featured as a FIWARE-ready IoT Device, you need to follow some steps, including a simple validation of your
+product (implementation), where you will connect it to one of the FIWARE public instances available and you will have to
+perform the following tests:
 
 -   IoT services creation (optional)
 -   Device registration
 -   Obtain the list of devices (optional)
 -   Send measurements
 -   Read measurements (optional)
 
-Second step of the validation process requires you to fill in a template to describe how your hardware device
-communicates with FIWARE.
-
-This validation process is an auto evaluation process but it is assisted by FIWARE-ready IoT Devices Program reviewers
-(IoTReady@lists.fiware.org) who will review your tests logs and evaluate whether your device is connecting with the
-public instance correctly.
-
-Once you have successfully completed the FIWARE-ready IoT Devices Program, your device will be published on
-[FIWARE Marketplace](http://marketplace.fiware.org​).
+This validation process is an auto-evaluation process but it is assisted by FIWARE-ready IoT Device reviewers
+[mailto:IoTReady@lists.fiware.org](IoTReady@lists.fiware.org), who will review your tests logs and evaluate whether your
+device is connecting with the public instance correctly. Once you have successfully completed the FIWARE-ready IoT
+Devices Program, your device will be published on `http://marketplace.fiware.org`.
```

Comment:
your device will be published on the FIWARE-ready IoT devices section in the [FIWARE Marketplace](https://marketplace.fiware.org/pages/devices).

Reply:
fixed  2e0c06a